spec/board_spec.rb

```diff
@@ -48,6 +59,7 @@
       expect(row5_syms).to eq(['.', '.', '.', '.', '.', '.', '.'])
       expect(row6_syms).to eq(['.', '.', '.', '.', '.', '.', '.'])
     end
```

Comment:
After we "expect" an empty board, this test could become more widespread by creating a player, calling add_piece, and then "expect"-ing print_board to include the updated space